src/types/index.d.ts

```diff
@@ -231,3 +231,8 @@ export interface ItemTracked {
   itemId?: string;
   variationId?: string;
 }
+
+export interface ItemTrackedPurchase extends ItemTracked {
+  count?: number;
+  price?: number;
```

Comment:
The attributes here were defined as required on the Node SDK. Is there any reason we're making them optional here? https://github.com/Constructor-io/constructorio-node/commit/4e73d415c225aab4698f1b0c10ea4f584490b245#diff-d91c72b0c4408f78487da8710045c189275b3aee8df371b01b5ad358056f0255R342

Reply:
Well, the ticket made a request to ensure retrocompatibility.  I figured this was the best way to achieve that since required parameters would break existing implementations.
Not sure how much of that is relevant for this particular client though, I´m happy to correct it if you think it´s better.

Reply:
After a discussion with Islam, we will make these properties optional in the Node repo as well. I´ll open a PR shortly.